modules/hardware/laptop.nix

```diff
@@ -35,6 +35,13 @@ in {
     # Disk configuration
     disko.devices.disk = hwDefinition.disks;
 
+    # Hardware specific kernel parameters
+    boot = {
+      initrd.availableKernelModules = ["nvme" "zfs"];
+      supportedFilesystems = ["zfs"];
+      kernelPackages = config.boot.zfs.package.latestCompatibleLinuxPackages;
```

Comment:
this section should be included in https://github.com/tiiuae/ghaf/blob/main/modules/hardware/x86_64-generic/x86_64-linux.nix#L27

Reply:
Done